message/validation/validation.go

```diff
@@ -76,7 +76,7 @@ type messageValidator struct {
 	index                   sync.Map
 	nodeStorage             operatorstorage.Storage
 	dutyStore               *dutystore.Store
-	ownOperatorID           spectypes.OperatorID
+	getOwnOperatorID        func() spectypes.OperatorID
```

Comment:
Suggesting we define type `OperatorIDFunc func() spectypes.OperatorID`
for `func() spectypes.OperatorID` due to its frequent use.
This would improve readability and ease future modifications. wdyt?

Reply:
I didn't do that because it's used in very different packages so I'm not sure what's best location for it